spec/factories/posts.rb

```diff
@@ -0,0 +1,6 @@
+FactoryBot.define do
+  factory :post do
+    title { "MyString" }
```

Comment:
We use a library `Faker` in production for false data so you could do `title { Faker::Lorem.word }` . . . I'm not sure if it's installed in Launchpad but you could try it, and even add it to the project as an improvement if it's not installed, if you want 😎

Reply:
Good to know! Looks like it's already in the Gemfile, so I'll play around with it in the next task when I write the tests